libs/auth/scopes/src/lib/tr.scope.ts

```diff
@@ -0,0 +1,3 @@
+export enum TRScope {
+  disabilityScope = '@tr.is/oryrki:read',
```

Comment:
Where is this used?

Reply:
It's not used in this PR, I'll remove it. 
This PR was split up and the TR client put in its own PR so that I could use it as well and I guess it got left over https://github.com/island-is/island.is/pull/8691